src/middleware/authJWT.js

```diff
@@ -0,0 +1,31 @@
+const jwt = require('jsonwebtoken');
+const usersData = require("../data/users.json");
+
+const verifyToken = (req, res, next) => {
+    try {
+        if (req.headers && req.headers.authorization && req.headers.authorization.split(' ')[0] === 'JWT') {
+            jwt.verify(req.headers.authorization.split(' ')[1], process.env.AUTH_SECRET, function (err, decode) {
```

Comment:
It's more readable to store `req.headers.authorization.split(' ')[1]` in a variable and use the variable in other places.